mix.exs

```diff
@@ -8,7 +8,7 @@ defmodule Decorator.Mixfile do
     [
       app: :decorator,
       version: @version,
-      elixir: "~> 1.5",
+      elixir: "~> 1.11",
       elixirc_options: elixirrc_options(Mix.env()),
```

Comment:
Minimum supported stable release with security patches is 1.11, see https://hexdocs.pm/elixir/1.15.7/compatibility-and-deprecations.html